provisioning/plugins/grafana-llm-app.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: 1
+
+apps:
+  - type: grafana-llm-app
+    jsonData:
+      openAIUrl: https://api.openai.com
+      embeddings:
+        type: openai
+        openai:
+          url: http://vectorapi:8889
+      vectorStore:
+        type: grafana/vectorapi
+        grafanaVectorAPI:
+          url: http://vectorapi:8889
+        collections:
+          - name: grafana:core:dashboards
```

Comment:
When writing #43 I noticed that qdrant doesn't like a bunch of characters in collection names 🤦 so maybe we should change this?

Although they also [recommend creating _one_ collection](https://qdrant.tech/documentation/faq/qdrant-fundamentals/#how-many-collections-can-i-create) and [using payload-based partitioning](https://qdrant.tech/documentation/tutorials/multiple-partitions/) so perhaps it's not a big deal, and I should change that PR to do so...

Reply:
Fixed this in 0bb2a5f, we can switch to a single collection later if we want. That _would_ let us do cool things like searching for multiple types of object matching the same search field, which would be interesting.